tests/test_properties.py

```diff
@@ -0,0 +1,31 @@
+from honeybee_schema.radiance.properties import ApertureRadiancePropertiesAbridged, \
+    DoorRadiancePropertiesAbridged, FaceRadiancePropertiesAbridged, \
+    ShadeRadiancePropertiesAbridged, RoomRadiancePropertiesAbridged, \
+    ModelRadianceProperties
+
+import os
+import json
+
+# target folder where all of the samples live
+root = os.path.dirname(os.path.dirname(__file__))
+target_folder = os.path.join(root, 'samples', 'model')
+
+
+file_path = os.path.join(target_folder, 'model_complete_office_floor.json')
+with open(file_path) as json_file:
+    office_model = json.load(json_file)
+
+
+def test_model_radiance_properties():
+    model_rad_props = office_model['properties']['radiance']
+    ModelRadianceProperties.parse_obj(model_rad_props)
```

Comment:
These tests really aren't necessary since these samples are all being checked in the `test_model.py`. I guess it doesn't hurt to have them, though.

Reply:
Yeah, the only reason I added them is in scenarios where you want to isolate just properties for checking. Feel free to delete this later.